### ISSUE TITLE
Program does not work for cutoffs other than zero

### ISSUE THREAD

**Issue by sergiocorreia** [acarril/rddsga#2]
It seems that when the bandwidth criteria is set ([here](https://github.com/acarril/rddsga/blob/07c828582bdf730fe3ac48f66d203af78b2e4b09/rddsga.ado#L132)), it ignores the cutoff:

```
local bwidth abs(`assignvar') < `bwidth'
```

The fix is probably to set:

```
local bwidth abs(`assignvar'-`cutoff') < `bwidth'
```

Otherwise, nothing happens when I run the command (which is tracked to the regress/ivregress calls dropping the variables of interest)

**Comment by acarril** [comment]
I think you are right. When I coded this I always assumed a 0 cutoff. I'll fix it today, thank you for the suggestion.

**Comment by acarril** [comment]
Fixed in 02a30b9
